### PR TITLE
bioperl-1.7.0

### PR DESCRIPTION
**Pull request by martin-mann**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/perl-inline-c/bld.bat [hunk 1]
```diff
@@ -0,0 +1,29 @@
+:: If it has Build.PL use that, otherwise use Makefile.PL
+IF exist Build.PL (
+    perl Build.PL
+    IF errorlevel 1 exit 1
+    Build
```
Comment (bgruening) [inline recipes/perl-inline-c/bld.bat]: @martin-mann can you remove all the *.bat files they are not used.

**Comment by bgruening** [comment]
`perl-sharedir-install` is missing I think. But even worse `Can't locate Parse/RecDescent.pm in @INC` and I do not know why :(

**Comment by bgruening** [comment]
@martin-mann how do you generate these recipes? Are you using the skeleton generator? If so why does it include old versions?

**Comment by martin-mann** [comment]
any idea for this error in [travis build](https://travis-ci.org/bioconda/bioconda-recipes/jobs/172272785)?

line 882:

```
+ set -o pipefail

+ export PERL_MM_USE_DEFAULT=1

+ PERL_MM_USE_DEFAULT=1

+ '[' -f Build.PL ']'

+ perl Build.PL

+ tee configure.log

Do you want to run the Bio::DB::GFF or Bio::DB::SeqFeature::Store live database tests? y/n [n]   - will not run the BioDBGFF or BioDBSeqFeature live database tests

Install [a]ll BioPerl scripts, [n]one, or choose groups [i]nteractively? [a]   - will install all scripts

Do you want to run tests that require connection to servers across the internet

(likely to cause some failures)? y/n [n]   - will not run internet-requiring tests

Created MYMETA.yml and MYMETA.json

Creating new 'Build' script for 'BioPerl' version '1.007000'

+ ./Build

/opt/conda/conda-bld/work/BioPerl-1.007000/conda_build.sh: ./Build: /opt/conda/conda-bld/_b_env_placehold_placehold_placehold_placehold_placehold: bad interpreter: No such file or director
```

**Comment by martin-mann** [comment]
wie gesagt: ICH HAB KEINE AHNUNG! sprich ich schraub die yaml files per 
HAND! krass oder? voll ätzend oder?

nun denn.. brauch da wohl mal ne einführung.. ;)



Am 01.11.2016 um 11:45 schrieb Björn Grüning:
> @martin-mann how do you generate these recipes? Are you using the skeleton generator? If so why does it include old versions?
>

**Comment by bgruening** [comment]
Travis hangs, lets close/repoen this PR.

**Pull request closed without merging**